internal/user/repository/eventsourcing/eventstore.go

```diff
@@ -317,6 +325,39 @@ func (es *UserEventstore) InitCodeSent(ctx context.Context, userID string) error
 	return nil
 }
 
+func (es *UserEventstore) VerifyInitCode(ctx context.Context, userID, verificationCode, password string) error {
+	if userID == "" || verificationCode == "" {
+		return caos_errs.ThrowPreconditionFailed(nil, "EVENT-lo9fd", "userId or Code empty")
+	}
+	//TODO: Check PW Policy
```

Comment:
we have to implement it now

Reply:
pw complexity policy is already implemented, if we use hashifexisting on Password model, it will be checked